train model.py

```diff
@@ -0,0 +1,93 @@
+from model import CNNtoRNN
+from downloader import get_coco_loader
+from tools import save_checkpoint, load_checkpoint, bert_cos_loss
+import torch
+from pytorch_pretrained_bert import BertModel
+from torch import nn
+import torch.optim as optim
+from tqdm import tqdm
+
+# creating data_loaders
+data_loader, val_loader, dataset = get_coco_loader()
```

Comment:
It's preferably, if you can put all the logic to `main()` function and run it under the `if __name__ == "__main__":` construction.

Also adding an ArgumentParser here is a good idea